### PR TITLE
remove client.py

### PR DESCRIPTION
**Pull request by mxyng**
See [ollama-python](https://github.com/jmorganca/ollama-python)

### CODE REVIEW

**Comment by pdevine** [comment]
Looks fine. You should probably stick a note in the README and the docs about how to get the official python client.

File: README.md [hunk 1]
```diff
@@ -248,6 +248,10 @@ curl http://localhost:11434/api/chat -d '{
 
 See the [API documentation](./docs/api.md) for all endpoints.
 
+## Official Integrations
```
Comment (jmorganca) [inline README.md:251]:
```suggestion
## Integrations
```

**Comment by jmorganca** [comment]
small nit otherwise LGTM! Thanks!!

File: README.md [hunk 2]
```diff
@@ -248,6 +248,10 @@ curl http://localhost:11434/api/chat -d '{
 
 See the [API documentation](./docs/api.md) for all endpoints.
 
+## Integrations
```
Comment (BruceMacD) [inline README.md:251]:
```suggestion
## Official Integrations
```
just a suggestion

**Review by BruceMacD — APPROVED** [review]
nit suggestion, but lgtm

**Pull request merged**